Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```
@@ -128,6 +128,8 @@
   - type: GunRequiresWield #remove when inaccuracy on spreads is fixed
   - type: Gun
     fireRate: 3
+  - type: GunSpreadModifier # Sunrise-add #Port WZ #37731
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)